### ISSUE TITLE
Can isDeviceConnected detect devices connected through the OS Bluetooth?

### ISSUE THREAD

**Issue by jkpvecino** [Polidea/react-native-ble-plx#380]
I'm currently trying to see if the package can detect if a device that's currently connected through the OS's Bluetooth module is also seen as connected within the app.  Essentially I'm doing the following things:

1. Connecting the device through the Android OS's Bluetooth functionality
2. Calling isDeviceConnected on the device_id to see if it's connected.

When I call startDeviceScan and I scan for the device, it comes up, but when I call isDeviceConnected or device.isConnected, I receive a False. Does that mean that even if a Bluetooth device is connected through the OS, the module will not recognize the device as connected?

**Comment by jclrosa** [comment]
On my case, I connected to a BLE device through the IOS Bluetooth menu (outside the app context), then when I enter the app, I need to know which is the device I am connected to. 

When I do what I described above, the connected device does not show on the scanAndConnect method, which is expectable, since the device is not `available` anymore. However I would like to know what is the device that I am connected to.

I tried `connectedDevices()`, without passing any device_id on the arguments array, because I don’t have any device_id to compare to, but the answer is always empty.  I just need to know what is the device that I am connected if this is the case. Is there any way to achieve it?
